src/Notes/NavigationNudge.php

```diff
@@ -33,13 +33,38 @@ public function __construct() {
 		add_action( 'update_option_' . Navigation::TOGGLE_OPTION_NAME, array( $this, 'action_note' ), 10, 2 );
 	}
 
+	/**
+	 * Should this note exist? (The navigation feature should exist.)
+	 */
+	public static function should_note_exist() {
```

Comment:
Nice!  This is not a bad name by any means, but there may be opportunity to improve.  I'm not sure I can think of a better one at the moment though.

`is_applicable` maybe?

Not a blocker for this PR; we can always iterate on this.